sys/include/net/gnrc/tcp.h

```diff
@@ -34,13 +34,62 @@
 extern "C" {
 #endif
 
+
+/**
+ * @brief Address information for a single TCP connection endpoint.
+ * @extends sock_tcp_ep_t
+ */
+typedef struct {
+    int family;                            /**< IP Address family. */
```

Comment:
As far as I'm aware "IP Address" is not a proper noun, so

```suggestion
    int family;                            /**< IP address family. */
```

Same below

Reply:
Done